Tests/NIOTests/IntegerBitPackingTests.swift

```diff
@@ -0,0 +1,55 @@
+//===----------------------------------------------------------------------===//
+//
+// This source file is part of the SwiftNIO open source project
+//
+// Copyright (c) 2021 Apple Inc. and the SwiftNIO project authors
+// Licensed under Apache License v2.0
+//
+// See LICENSE.txt for license information
+// See CONTRIBUTORS.txt for the list of SwiftNIO project authors
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+//===----------------------------------------------------------------------===//
+
+import XCTest
+@testable import NIO
+
+final class IntegerBitPackingTests: XCTestCase {
+    func testAllUInt8PairsRoundtrip() {
+        for i in UInt16.min ... UInt16.max {
+            let unpacked = IntegerBitPacking.unpackUInt8UInt8(i)
+            XCTAssertEqual(i, IntegerBitPacking.packUInt8UInt8(left: unpacked.0, right: unpacked.1))
+        }
+    }
+
+    func testExtremesWorkForUInt32UInt16UInt8() {
+        XCTAssert((.max, .max, .max) ==
+                    IntegerBitPacking.unpackUInt32UInt16UInt8(IntegerBitPacking.packUInt32UInt16UInt8(left: .max,
```

Comment:
@Lukasa / @glbrntt this is the new API now with the `left:middle:right:` labels. I'm okay with it but I think no labels is nicer. I don't think it adds any value, just makes it harder to read. But it's not API so I'm cool with it. Looks like Lua arrays :P

Reply:
@weissi I was only asking for labels on the generic `packUU` and `packUUU` functions!

You absolutely don't need them on the `packUIntXUIntY` functions.